src/styles/pagination.scss

```diff
@@ -7,94 +7,94 @@
 
 $block: #{$fd-namespace}-pagination;
 
-.#{$block} {
+@mixin fd-pagination-mobile() {
+  .#{$block}__per-page + .#{$block}__nav {
+    @include fd-set-margins-x('auto', 0);
+  }
+
+  .#{$block}__total,
+  .#{$block}__per-page-label,
+  .#{$block}__link,
+  .#{$block}__more {
+    display: none;
+  }
 
-  $fd-pagination-link-padding-x: 0.25rem;
-  $fd-pagination-padding-x: 0.5rem;
+  .#{$block}__button--mobile,
+  .#{$block}__label {
+    display: flex;
+  }
+}
 
+.#{$block} {
   @include fd-reset();
+  @include fd-flex-between();
 
   font-size: var(--sapFontSize);
+  padding: 0 1rem;
+  height: 2.75rem;
+  width: 100%;
 
-  &__total {
+  .#{$block}__per-page {
```

Comment:
Okay, i reverted for pagination-only elements but left for buttons, labels, inputs as we're using appropriate components and there may be issues like `.fd-button` styles overwrite `.fd-pagination__link` styles.

Also left in `fd-pagination-mobile` mixin as it can be applied to `.fd-pagination--mobile` and then we will have for example `.fd-pagination--mobile__link` but we don't have such element.